runtime/vm/guardedstorage.c

```diff
@@ -48,6 +49,15 @@ j9gs_initializeThread(J9VMThread *vmThread)
 	if (IS_THREAD_GS_INITIALIZED(vmThread)) {
 		success = 1;
 	} else {
+
```

Comment:
Please remove this empty line (here and below). Empty lines can be used to improve code readability however in this case it is not necessary - tab is enough

Reply:
fixed